receiver/signalfxreceiver/receiver_test.go

```diff
@@ -578,7 +580,7 @@ func Test_sfxReceiver_TLS(t *testing.T) {
 
 	url := fmt.Sprintf("https://%s/v2/datapoint", addr)
 
-	req, err := http.NewRequest("POST", url, bytes.NewReader(body))
+	req, err := http.NewRequestWithContext(context.Background(), "POST", url, bytes.NewReader(body))
```

Comment:
According to the [`NewRequest` docs](https://pkg.go.dev/net/http#NewRequest), `NewRequest` just wraps `NewRequestWithContext` and passes `context.Background`, so I think it would make sense to just leave this call as-is.

Reply:
```suggestion
	req, err := http.NewRequest("POST", url, bytes.NewReader(body))
```

Reply:
Reverting, thanks :)